cmd/manager/main.go

```diff
@@ -128,8 +135,27 @@ func main() {
 		os.Exit(1)
 	}
 
+	disableHTTP2 := func(config *tls.Config) {
```

Comment:
Is this one for test clusters or means for production clusters also?

Reply:
Production clusters as well. As of now, `curl` misses support for HTTP2 on some distros. We can enable it by default once we are sure that all the clients support HTTP2.

Kubebuilder's default scaffolding also follows the same procedure as of now.